Dockerfile

```diff
@@ -23,10 +23,21 @@ ENV MATTERWICK=/matterwick/matterwick \
 WORKDIR /matterwick/
 
 RUN  apk update && apk add ca-certificates
+ENV KUBECONFIG=/.kube/config
```

Comment:
using the code i share with you in the stack overflow we don't need this and only need the aws keys. 
I will send to you a sample code that I use to test

with that we don't need awscli and kubectl